meetings/osf_oauth2_adapter/migrations/0001_add_site_and_social_app.py

```diff
@@ -7,11 +7,11 @@
 from allauth.socialaccount.models import SocialApp
 from django.contrib.sites.models import Site
 
-from osf_oauth2_adapter.apps import OsfOauth2AdapterConfig
-from django.conf import settings
+from meetings.utils import OsfOauth2AdapterConfig
+import meetings.settings.local as local
 
-CLIENT_ID = settings.CLIENT_ID
-CLIENT_SECRET = settings.CLIENT_SECRET
+CLIENT_ID = local.CLIENT_ID
```

Comment:
This is not the way to handle `local.py`. Please look at how the `__init__.py` file works in the settings folder.